packages/lesswrong/server/ckEditor/ckEditorWebhook.ts

```diff
@@ -99,20 +153,52 @@ async function handleCkEditorWebhook(message: any) {
     case "comment.removed":
     case "commentthread.removed":
     case "commentthread.restored":
+    case "collaboration.user.connected": {
+      const userConnectedPayload = payload as CkEditorUserConnectionChange;
+      const userId = userConnectedPayload?.user?.id;
+      const documentId = userConnectedPayload?.document?.id;
+
+      const result = await createMutator({
+        collection: CkEditorUserSessions,
+        document: {
+          userId,
+          documentId
+        },
+        validate: false,
```

Comment:
I think we should not default to passing in `validate: false` except in unusual circumstances where we have no choice (such as creating new posts, because of the `contents` field type).